Index: GKlib/gk_arch.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- GKlib/gk_arch.h	(revision )
+++ GKlib/gk_arch.h	(revision )
@@ -59,13 +59,17 @@
 #endif
 
 #ifdef __MSC__
-/* MSC does not have rint() function */
-#define rint(x) ((int)((x)+0.5))  
+/* MSC may not have rint() function */
+#if(_MSC_VER < 1800)
+#define rint(x) ((int)((x)+0.5))
+#endif
 
 /* MSC does not have INFINITY defined */
 #ifndef INFINITY
+#if(_MSC_VER < 1900)
 #define INFINITY FLT_MAX
 #endif
 #endif
+#endif
 
 #endif
